docs/dxp/7.x/en/site-building/site-appearance/site-appearance.md

```diff
@@ -12,46 +12,33 @@ The design and functionality of a Liferay site can be adjusted and customized th
 
```

Comment:
We may want to use "Master Pages" or "Master Page Templates" rather than "Page Templates", as the latter has a way older meaning in liferay. What matters here is the 'master' term

Reply:
I can change that.

To be clear, though -- does this mean that these things would only apply to Master Page Templates though, and not regular Page Templates?

Reply:
Being it true that page templates offer a way to add elements to a page made from it, I would say that we never introduced them as a customization mechanism like master pages + style books are being introduced in 7.3.

We may want, nevertheless, to mention the concept of page template (and link to the corresponding articles for content and widget page templates) within the Master Page Templates section, to help reader to differentiate bot things, yet keeping the intent of presenting the pack "Masters + stylebooks" in a more appealing and cohesive way.